examples/Tools/Factory_Set_Initial_Servo_Position/Factory_Set_Initial_Servo_Position.ino

```diff
@@ -49,7 +49,7 @@ static std::array<InitialServoPosition, 6> const INITIAL_SERVO_POSITION =
 
 bool set_initial_servo_position(int const id, float const target_angle)
 {
-  auto isTimeout = [](unsigned long const start) -> bool { return ((millis() - start) > 2000); };
+  auto isTimeout = [](unsigned long const start) -> bool { return ((millis() - start) > 5000); };
```

Comment:
It doesn't harm any operation, you just have to wait longer in case something does indeed time out :shrug: .

Reply:
Come to think of it, you are miss reading the code. The timeout function is only applied for 2 individual steps. This means that the timeout is not for the whole function but only for those two steps. Consequently the `delay(500);` is not negatively effecting the timeout.

Can we please go back to assessing a root cause before throwing around random "hot fixes"?